application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -102,6 +122,34 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
             .insert());
     }
 
+    private void handleListCommand(SlashCommandInteractionEvent event) {
+        BiFunction<Long, Instant, String> getDescription = (channelId, remindAt) -> """
+                Channel: %s
+                Remind at: %s""".formatted(MessageUtils.mentionChannelById(channelId),
+                TimeFormat.DEFAULT.format(remindAt));
+
+        EmbedBuilder remindersEmbed = new EmbedBuilder().setTitle("Pending reminders")
+            .setColor(RemindRoutine.AMBIENT_COLOR);
+
+        long guildId = event.getGuild().getIdLong();
+        long userId = event.getUser().getIdLong();
+
+        database
+            .read(context -> context.selectFrom(PENDING_REMINDERS)
+                .where(PENDING_REMINDERS.GUILD_ID.eq(guildId)
+                    .and(PENDING_REMINDERS.AUTHOR_ID.eq(userId)))
+                .orderBy(PENDING_REMINDERS.CREATED_AT.asc())
+                .fetch())
+            .forEach(reminder -> remindersEmbed.addField(reminder.getContent(),
+                    getDescription.apply(reminder.getChannelId(), reminder.getRemindAt()), false));
```

Comment:
what happens if the reminder text is quite long? people can create really long reminders and this field-title for sure has some smaller limit, no? might have to use `MessageUtils.abbreviate`.

and if the limit is too small, using the title might not even be an option anymore. its important that people can read their full reminder here (in most cases).

that said, lets say one creates reminders of the size:
```java
i have to do foo and then bar, please tell me foo and bar. its important that i do foo and bar.
```
how many of those can be created until the embed runs out of its max size and we get exceptions? we might have to add pagination. or at least make sure that the UX is good if this happens, i.e. we have to limit the shown reminders based on total length or similar.

Reply:
the field title can hold 256 chars max. so in the worst case a field can have around 300 chars and a embed can have 4k smth chars at max so a embed should be able to hold 13-14 of very big reminders. ig the solution is to setup pagination with 10 reminders on each page

also this is what it'll look like with big reminders each is 250 chars
![image](https://user-images.githubusercontent.com/73871477/198113111-a991b42e-f4d4-4866-9748-4bf95a5d5339.png)
ui is not a problem will need to use the abbreviate method on longer reminders

Reply:
Enjoy adding pagination! Its annoying :D

Reply:
you can take a look at my PR to see how i implemented it

Reply:
> Enjoy adding pagination! Its annoying :D

lol yea, but i've did that before on this repo shouldn't be too hard

Reply:
Hm im interested how you did it

Reply:
we can think about adding some helpers for pagination, if u had trouble with it. seems to be sth that we will frequently need.

Reply:
> we can think about adding some helpers for pagination, if u had trouble with it. seems to be sth that we will frequently need.

Cool idea. Yes it would really make sense and make individual features a lot lighter. I may do that after the bookmarks feature gets merged.